### PR TITLE
Members readme is in a private `.github-private` repository

### PR DESCRIPTION
**Pull request by thispsj**
### Why:

Closes nothing as such. For reference, refer to #17257 

### What's being changed:

Just a change from <kbd>public</kbd> to <kbd>private</kbd>

Affects [this document](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile#adding-a-member-only-organization-profile-readme).

### Check off the following:

- [X] I have reviewed my changes in staging.
- [X] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@thispsj Thank you for opening this draft PR. Please leave a comment when the PR is ready for a review so I will get a notification to triage it ✨

**Comment by ramyaparimi** [comment]
@thispsj Ops! That comment was supposed to be on the other draft PR!! Please comment on the other draft PR when it is ready for ready. 

I will get this PR triaged for a review 👀.

File: content/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ You can format text and include emoji, images, and GIFs in your organization pro
 
 ## Adding a member-only organization profile README
 
-1. If your organization does not already have a `.github-private` repository, create a public `.github-private` repository. This action can be performed by any user that has write access to that repository.
+1. If your organization does not already have a `.github-private` repository, create a private `.github-private` repository. This action can be performed by any user that has write access to that repository.
```
Comment (lecoursen) [inline content/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile.md:52]:
```suggestion
1. If your organization does not already have a `.github-private` repository, create a private `.github-private` repository. 
```

**Review by lecoursen — DISMISSED** [review]
Thanks @thispsj! I added a suggestion to change something completely unrelated to your PR, just something I noticed while I was reading that line anyway! I'll apply that and then merge.

File: content/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ You can format text and include emoji, images, and GIFs in your organization pro
 
 ## Adding a member-only organization profile README
 
-1. If your organization does not already have a `.github-private` repository, create a public `.github-private` repository. This action can be performed by any user that has write access to that repository.
+1. If your organization does not already have a `.github-private` repository, create a private `.github-private` repository.
```
Comment (lecoursen) [inline content/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile.md:52]:
```suggestion
1. If your organization does not already have a `.github-private` repository, create a private repository called `.github-private`. 
```

**Review by lecoursen — APPROVED** [review]

**Pull request merged**